src/OSPSuite.CLI.Core/MinimalImplementations/RGroupRepository.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections.Concurrent;
+using System.Collections.Generic;
+using OSPSuite.Core.Domain;
+
+namespace OSPSuite.CLI.Core.MinimalImplementations
+{
+    public class RGroupRepository : IGroupRepository
```

Comment:
same with this. Why a default naming comvention than other MinimalImplementations?